lib/markdown-service.js

```diff
@@ -66,8 +66,11 @@ class MarkdownService {
             data = data.replace(/<strong>|<\/strong>/g, '**');
 
             // add escape for numbered lists
-            // https://regex101.com/r/nB8bV3/1
-            data = data.replace(/(^\d{1,2})(. )/, '$1\\\\. ');
+            // https://regex101.com/r/SO5haw/1
+            if (data.match(/^\d{1,4}\./)) {
+                // https://regex101.com/r/nB8bV3/4
+                data = data.replace(/(^\d{1,4})(.)/, '$1\\$2');
```

Comment:
The if statement with `data.match(/^\d{1,4}\./)` should keep any incoming data from hitting this regex unless it is a number followed by a ".". 

In testing, I cannot get `1a.` to enter into this if statement. (I added this if statement because in previous testing, I noticed it was possible to have numerals in places we wouldn't want to necessarily format. The intent of the if statement is to only allow 1-4 digit numberals followed by a "." to be processed using `data.replace(/(^\d{1,4})(.)/, '$1\\$2')`.)

We only want to format if it  is a numeral followed by a "." Right?

Is the intent to remove this if statement and have a standalone regex replace?
https://regex101.com/r/SO5haw/1/tests